packages/send/frontend/src/stores/auth-store.ts

```diff
@@ -73,28 +70,36 @@ export const useAuthStore = defineStore('auth', () => {
    */
   async function handleOIDCCallback() {
     try {
-
       const user = await userManager.signinCallback();
       currentUser.value = user;
 
-      window.addEventListener("message", (e) => {
-        if (e.origin === window.location.origin && e.data?.type === "TB/BRIDGE_READY") {
-          console.log("[web app] bridge says: ready");
+      window.addEventListener('message', (e) => {
+        if (
+          e.origin === window.location.origin &&
+          e.data?.type === 'TB/BRIDGE_READY'
+        ) {
+          console.log('[web app] bridge says: ready');
         }
       });
 
       // Send one tiny ping to the bridge.
-      window.postMessage({ type: "APP/PING", text: "hello from auth store 👋" }, window.location.origin);
+      window.postMessage(
```

Comment:
is this ping necessary? If so, could you add an explanation in the comment? Since I'm not sure why we need to send this